deploy/olm-catalog/splunk/1.0.2/splunk.v1.0.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,349 @@
+      deployments:
+      - name: splunk-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: splunk-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: splunk-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: splunk-operator
+                - name: RELATED_IMAGE_SPLUNK_ENTERPRISE
+                  value: docker.io/splunk/splunk:8.2.0
+                image: docker.io/splunk/splunk-operator:1.0.2
+                imagePullPolicy: IfNotPresent
+                name: splunk-operator
+                resources: {}
+              serviceAccountName: splunk-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - configmaps
+          - secrets
+          - pods
+          - pods/exec
+          - serviceaccounts
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - enterprise.splunk.com
+          resources:
+          - '*'
+          - indexerclusters
+          - clustermasters
+          - licensemasters
+          - searchheadclusters
+          - standalones
+          verbs:
+          - '*'
+        serviceAccountName: splunk-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - Splunk
+  - Enterprise
+  - SearchHeadCluster
+  - IndexerCluster
+  - Standalone
+  - LicenceMaster
+  - ClusterMaster
+  - Deployer
+  links:
+  - name: Splunk
+    url: https://splunk.domain
+  maintainers:
+  - email: smohan@splunk.com
+    name: '''Sirish Mohan'
+  - email: sgontla@splunk.com'
+    name: Subba Gontla
```

Comment:
Didn't think that our names come here :-)
@smohan-splunk , just noticed, following is what is on develop branch. hope that is the same info we need to use here.

```  keywords:
  - Splunk
  - logging
  links:
  - name: Splunk
    url: https://splunk.domain
  maintainers:
  - email: docker-maint@splunk.com
    name: Splunk Docker Maintainers